packages/sd-jwt/src/__tests__/sd-jwt.test.ts

```diff
@@ -0,0 +1,346 @@
+import { subtle } from 'node:crypto'
```

Comment:
these first 2 dependencies imported in this file could present problems. I know this is only a test, but we try to only use dependencies that work across all supported platforms. Is it not possible to use one of the other crypto libraries already used elsewhere for this?

Reply:
I think we can, It's just the verify function

Reply:
Have you ever written a verify function that is platform-independent? I tried to use the @noble package, but it doesn’t work.